bank-sdk/sdk/src/doc/source/migrating-from-gvl.rst

```diff
@@ -157,23 +159,21 @@ This is how you need to use it with the Gini Bank SDK:
 Component API
 -------------
 
-For the Component API you will interact with the Gini Capture SDK directly.
-
-The most important changes compared to the Gini Vision Library are:
-
-* deprecated code was removed,
-* support for native activities and fragments was dropped,
-* package was renamed from ``net.gini.android.vision.*`` to ``net.gini.android.capture.*``,
-* ``GiniVision`` in class or interface names was renamed to ``GiniCapture``.
+The Component API allowed more UI customisation options for the cost of a more difficult integration and maintenance. It
+was based on fragments and you had to manage navigation between them and also update the navigation whenever we introduced
+breaking changes.
 
-If you are already using Jetpack, then migrating the Component API should be fairly straight forward. In case you are
-using native activities and fragments, then please first switch to using the Jetpack ``AppCompatActivity``.
+Maintaining the Component API along the simpler Screen API required an increasing amount of effort as we added new
```

Comment:
Looks good 😊 I think you've missed _with_ in Maintaining the Component API **along with** the simpler Screen API.